dev/TabView/TabView.xaml

```diff
@@ -159,13 +123,82 @@
+                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
+                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
+                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
+                                Background="Transparent"
+                                BorderThickness="1"
+                                Delay="50"
+                                FontFamily="Segoe MDL2 Assets"
+                                Interval="100"
+                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                Visibility="Collapsed"
+                                Content="&#xE76B;"/>
+
+                            <ScrollContentPresenter x:Name="ScrollContentPresenter"
+                                Grid.Column="1"
+                                Padding="{TemplateBinding Padding}"
+                                contract4Present:TabFocusNavigation="Once" />
+
+                            <RepeatButton x:Name="ScrollIncreaseButton"
+                                Grid.Column="2"
+                                VerticalAlignment="Bottom"
+                                HorizontalAlignment="Center"
+                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
+                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
+                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
+                                Background="Transparent"
+                                BorderThickness="1"
+                                Delay="50"
+                                FontFamily="Segoe MDL2 Assets"
+                                Interval="100"
+                                Style="{StaticResource RepeatButtonRevealStyle}"
+                                Visibility="Collapsed"
+                                Content="&#xE76C;"/>
+                        </Grid>
+                    </Border>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Name="TabViewButtonStyle" TargetType="Button">
+        <Setter Property="FontSize" Value="12"/>
+        <Setter Property="FontFamily" Value="Segoe MDL2 Assets"/>
+        <Setter Property="Width" Value="24"/>
+        <Setter Property="Height" Value="24"/>
+        <Setter Property="VerticalAlignment" Value="Bottom"/>
+        <Setter Property="Margin" Value="4,0,4,3"/>
+        <Setter Property="Padding" Value="0"/>
+        <Setter Property="Background" Value="Transparent"/>
+    </Style>
+
     <Style TargetType="local:TabViewItem">
         <Setter Property="HorizontalContentAlignment" Value="Left"/>
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TabViewItem">
 
                     <Grid
+                        x:Name="BackgroundGrid"
```

Comment:
Is this name used anywhere?  Ctrl+F is failing me if so.

Reply:
No, turns out I'm not using it anymore. Removed.